src/components/connection_handler/include/connection_handler/connection_handler_impl.h

```diff
@@ -660,6 +670,16 @@ class ConnectionHandlerImpl
   Connection* GetPrimaryConnection(
       const ConnectionHandle connection_handle) const;
 
+  /**
+   * @brief Handling a reason for closing a session
+
+   * @param connection_id Connection unique identifier.
+   * @return the reason of connection closed,if connection processed final
+   message, returns kFinalMessage,otherwise returns kCommon.
```

Comment:
```suggestion
   * @brief Handling a reason for closing a session
   * @param connection_id Connection unique identifier.
   * @return the reason for connection closed, if connection processed final 
   message returns kFinalMessage, otherwise returns kCommon.
```

Reply:
@LitvinenkoIra 
Fixed in:https://github.com/LuxoftSDL/sdl_core/pull/136/commits/56d01fc91f5af95f609d1a7d3d91e73648eacb97